autoblocks/_impl/testing/run.py

```diff
@@ -300,7 +250,16 @@ def run_test_suite(
     # How many evaluators to run concurrently on the result of a test case
     max_evaluator_concurrency: int = 5,
 ):
-    init_global_vars_and_event_loop()
+    cli_server_address = AutoblocksEnvVar.CLI_SERVER_ADDRESS.get()
+    if not cli_server_address:
+        raise RuntimeError(
+            "Autoblocks tests must be run within the context of the testing CLI.\n"
+            "Make sure you are running your test command with:\n"
+            "$ npx autoblocks testing exec -- <your test command>"
+        )
+
+    if not event_loop_started():
+        init_event_loop(client=httpx.AsyncClient(base_url=cli_server_address))
```

Comment:
you're going to need to drop this arg and refactor this to pass the base_url on every call

Reply:
alternatively you could ignore the global client for the testing sdk and just use your own global client like it was setup previously. probably not super important for the testing sdk to use the package-wide global client

Reply:
I ended up reworking it so the run_test_suit looks like

```python
def run_test_suite(
    ...stuff...
):
    cli_server_address = AutoblocksEnvVar.CLI_SERVER_ADDRESS.get()
   ...stuff

    global_state.init()
    global_state.http_client().base_url = cli_server_address
    future = asyncio.run_coroutine_threadsafe(
        ...stuff..
        global_state.event_loop(),
    )
    future.result()
```